src/MicrosoftTeams.ts

```diff
@@ -830,6 +838,29 @@ namespace microsoftTeams {
     sendMessageRequest(parentWindow, "openFilePreview", params);
   }
 
+  export interface ShowNotificationParameters {
+    message: string;
+    isDownloadComplete: boolean;
+  }
+
+  /**
+   * @private
+   * Hide from docs.
+   * ------
+   * download file.
+   * @param file The file to download.
```

Comment:
Update the comments once you make this API more generic.

Reply:
yes